src/routes/blog/spyder-6-carlos/+page.md

```diff
@@ -0,0 +1,156 @@
+Additionally, we made the breakpoints table collapsible and fixed a theme incompatibility issue in the frames browser.
+
+![Before and after of Spyder's run preferences pane, on the Run Executors tab now showing a list of default run presets as well as a streamlined and more compact layout](run-preferences.png "Run preferences, showing Carlos' UI improvements")
+
+Finally, I built upon [Edgar Margffoy](https://github.com/andfoy) and [Stephannie Jimenez](https://github.com/steff456)’s new modular architecture for running files in Spyder 6, greatly [improving its UI and UX](https://github.com/spyder-ide/spyder/pull/22141).
+These changes added customizable default run presets for specific file types, making it easier to customize run behavior.
+In addition, the layout of that preference pane along with the per-file run config and the run config dialog are substantially simplified, cleaned up and polished, making the UI easier to use and more intuitive.
+
+![Before and after of Spyder's Run Config Per File dialog listing file execution settings, with the after version showing a simpler, collapsible and more refined layout](run-config-per-file.png "Carlos' UI improvements to the Run Config Per File dialog")
+
+
+## Insights going forward
+
+The Editor migration and the new remote client plugin were a fascinating set of stress tests for Spyder's plugin API, originally introduced in Spyder 5.
+They required implementing close integration with the IPython Console, but without affecting the already extensive connections between the console and several other plugins.
+
+Fortunately, our API passed the test really well, and we were able to have a working prototype of the Editor as a plugin for Linux and Mac in just four months.
+This means that we're in good shape to tackle the complex challenges that will come in 6.1 and beyond to add remote development capabilities to other Spyder plugins, such as the Editor and Projects.
+
+
+## What's next for Spyder?
+
+![Spyder-Env-Manager pane showing an environment selector and a list of installed package names, descriptions and versions as well as toolbar options to interact with them](spyder-env-manager-alpha.png "Working alpha version of the Spyder-Env-Manager plugin")
+
+Next year, I'll be working on more UI/UX improvements on several different fronts.
+
+I'll integrate the [Spyder-Env-Manager](https://github.com/spyder-ide/spyder-env-manager) plugin, developed by [Daniel Althviz](https://github.com/dalthviz) and [Juan-Sebsatian Bautista](https://github.com/jsbautista), with the remote development client.
+Spyder-Env-Manager allows you to create Conda environments as well as install and manage packages within them from within Spyder's graphical interface—no arcane commands or external GUIs, and mo separate install of Anaconda with our standalone installers.
```

Comment:
I kept the first change, but I modified the second to "needed" (missing from the original) since "will be necessary" does not fit within the grammar/phrasing in that specific context.

```suggestion
Spyder-Env-Manager allows you to create Conda environments as well as install and manage packages within them directly from Spyder—no arcane commands or external GUIs, and no separate install of Anaconda needed with our standalone installers.
```